dataStandard_versioned_docs/version-5/model-reference/student-health-domain/entities-references-descriptors.md

```diff
@@ -0,0 +1,58 @@
+| ------------- | ------------------------------------------------------------------------------------------------------------------------------------------- |
+| StudentHealth | This event entity represents actions taken by an education organization after a disruptive event that is recorded as a discipline incident. |
+
+## Extended References
+
+| Name                  | Description                                                                                                                                                                                                                                                                                                                                       |
+| --------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| EducationOrganization | This entity represents an educational organization.                                                                                                                                                                                                                                                                                               |
+| Student               | This entity represents an individual for whom instruction, services, and/or care are provided in an early childhood, elementary, or secondary educational program under the jurisdiction of a school, education agency or other institution or program. A student is a person who has been enrolled in a school or other educational institution. |
+
+## Student Health Domain Descriptors
+
+| Entity        | Name                            | Description                                                                                    | Usage Classification | EDFacts Mapping | Commonly Used | Commonly State-Defined |
+| ------------- | ------------------------------- | ---------------------------------------------------------------------------------------------- | -------------------- | --------------- | ------------- | ---------------------- |
+| StudentHealth | ImmunizationType                | An indication of the type of immunization that an individual has satisfactorily received.      | Standard             |                 | Yes           | Yes                    |
+| StudentHealth | NonMedicalImmunizationExemption | The type of nonmedical exemption from vaccination claimed by the student's parent or guardian. | Local                |                 | Yes           |                        |
+
+:::tip
+
+See [Non-Normative Descriptor Classifications](/reference/data-exchange/umd/non-normative-descriptor-classifications.md) for more information on _Usage Classification_.
+
+:::
```

Comment:
@chrisgocode  Most of the pages have a link "see [info]..." in the header of this table. I moved the link to be a tip _below_ the table, as seen above. Please update the other pages to use the same approach.